pkg/apis/fleet.cattle.io/v1alpha1/values.go

```diff
@@ -6,8 +6,20 @@ import (
 	"github.com/rancher/wrangler/pkg/data/convert"
 )
 
+// GlobalValues are values that are inserted into deployments containing some metadata about the deployment
+type GlobalValues struct {
+	Fleet *FleetGlobalValues `json:"fleet,omitempty"`
```

Comment:
Is the pointer here necessary? The consumption gets less error prone if we put this directly a struct.
```suggestion
	Fleet FleetGlobalValues `json:"fleet,omitempty"`
```

Reply:
Thanks! it looks much better now.